proto/frequenz/api/common/v1/microgrid/lifetime.proto

```diff
@@ -15,14 +15,27 @@ import "google/protobuf/timestamp.proto";
 // a microgrid asset, such as a component, connection, sensor, or any other
 // entity with a limited operational lifetime.
 //
-// !!! warning "Permanent Deletion"
-//     The `end_timestamp` indicates that the asset has been permanently removed
-//     from the system.
+// It is normally used to track the historical evolution of assets, and it
+// determines the validity period for the asset's data.
 //
+// If the `start_timestamp` is not set, the asset is considered to be in
+// operation since the beginning of the system's operational history.
+//
+// If the `end_timestamp` is not set, the asset is considered to be in
+// operation until the end of the system's operational history.
```

Comment:
The "end of history" part reads a bit odd, because history indicates past tense. I feel that phrase does not add anything here, and can be removed:
```suggestion
// If the `end_timestamp` is not set, the asset is considered to be in
// operation.
```

The same comment applies to other instances where end-timestamp is being documented.

Reply:
What about "indefinitely"? I would like to make it explicit that it is in operation now but also at any time in the future.

Reply:
Sounds good. Should then be "indefinitely _into_ the future".

Reply:
Updated.